parlai/tasks/wrapper/agents.py

```diff
@@ -18,43 +18,49 @@
 
 import copy
 
+from abc import ABC, abstractmethod
 from parlai.core.agents import create_agent_from_shared
 from parlai.core.opt import Opt
 from parlai.core.teachers import create_task_agent_from_taskname, Teacher
 
 
-class LabelToTextTeacher(Teacher):
+class AbstractWrapperTeacher(Teacher, ABC):
     """
-    Teacher that will shift message['labels'][0] into message['text'] for whatever task
-    is specified with --label-to-text-task.
-
-    Because the dialogue history is effectively overwritten by this action, all episodes
-    will be flattened into one example each.
+    Abstract teacher that will wrap around another teacher and allow for manipulating
+    the fields returned by the inner teacher.
     """
 
     @classmethod
-    def add_cmdline_args(cls, argparser):
-        # Teacher has no args
-        parser = argparser.add_argument_group('LabelToText args')
+    def add_cmdline_args(cls, parser):
+        parser = parser.add_argument_group('AbstractWrapper args')
         parser.add_argument(
-            '-l2tt',
-            '--label-to-text-task',
+            '-wt',
+            '--wrapper-task',
             type=str,
-            help='The task whose labels will get shifted into the text field',
+            help='The task which fields will be manipulated.',
```

Comment:
tiny nit: Maybe `'The task whose fields will be manipulated`?

Reply:
woops. fixed it